ansible-ipi-install/group_vars/all

```diff
@@ -0,0 +1,47 @@
+# The global variable file for IPI on Baremetal installation
+
+# The provisioning NIC (NIC1) used on all bm nodes
+prov_nic: eno1
+
+# The public NIC (NIC2) used on all bm nodes
+pub_nic: eno2
+
+# Activation-key for proper setup of subscription-manager
+activation_key: ""
+
+# Activation-key org_id for proper setup of subscription-manager
+org_id: ""
+
+# The directory used to store the cluster configuration files (install-config.yaml, pull-secret.txt, metal3-config.yaml)
+dir: /home/{{ myuser }}/clusterconfigs
+
+# Location of the pullsecret file on the remote provisioning bm node
+pullsecret_file: /home/{{ myuser }}/clusterconfigs/pull-secret.txt
+
+# Location of where the openshift-installer will be extracted
+extract_dir: /home/{{ myuser }}/extract
+
+# The user used to do the openshift-installation (requires passwordless sudo priv)
+myuser: ""
+
+# The cmd used for openshift-install extraction
+cmd: openshift-baremetal-install
+
+# The version of the openshift-installer 
+version: "4.3.0-0.nightly-2019-12-09-035405"
+
+# Provisioning IP address (default value)
+prov_ip: 172.22.0.3
+
+### Vars regarding install-config.yaml
```

Comment:
i think it's debatible whether we want to be rendering install-config which tends to be super specific for each user

Reply:
I think we can add additional logic for other options if required. However, I think having that file auto generated is a huge plus as creating it manually is a headache.

Reply:
yeah, as long as we have an escape hatch for providing your own install_config, I bet a lot of people will find autogenerating the install-config useful

Reply:
I could see that being a feature enhancement. I'd like not to necessary delay getting this playbook inside this repo.